app/routes/publications/overview/search.js

```diff
@@ -144,14 +171,8 @@ export default class PublicationFlowSearchRoute extends Route {
   }
 
   @action
-  loading(transition) {
-    // eslint-disable-next-line ember/no-controller-access-in-routes
-    const controller = this.controllerFor(this.routeName);
-    controller.isLoadingModel = true;
-    transition.promise.finally(() => {
-      controller.isLoadingModel = false;
-    });
-    return true;
+  loading() {
+    return false;
```

Comment:
same

Reply:
we do need to return false in this case because of the parent reload.
I have added the setting of the `isLoadingModel` so the content of the datable switches between loading and showing results.